jsapp/js/components/processing/analysis/analysisQuestions.reducer.ts

```diff
@@ -97,12 +105,17 @@ export const analysisQuestionsReducer: AnalysisQuestionReducerType = (
       return {
         ...state,
         isPending: false,
+        hasUnsavedWork: false,
         questions: action.payload.questions,
       };
     }
     case 'startEditingQuestion': {
       return {
         ...state,
+        // We can put this on the text input of titles and select
+        // questions for slight UX improvement but this is simpler
+        // and understandable.
```

Comment:
I don't understand this comment 😅

Reply:
So perhas:

> Instead of checking changes on every input in the edited question, we assume that, as soon as user starts to edit a question, there are unsaved changes. This is not ideal UX, but it's much simpler and still logical.